hooks/boot/02-message-helpers.rb

```diff
@@ -40,5 +40,15 @@ def new_install_message(kafo)
     def dev_new_install_message(kafo)
       say "      Initial credentials are <%= color('admin', :info) %> / <%= color('#{kafo.param('katello_devel', 'admin_password').value}', :info) %>"
     end
+
+    def failure_message
+      say "  <%= color('Something went wrong!', :bad) %> Check the log for ERROR-level output"
+    end
+
+    def log_message(kafo)
+      # This is always useful, success or fail
+      log = kafo.config.app[:log_dir] + '/' + kafo.config.app[:log_name]
```

Comment:
I think this also works:
```suggestion
      log = kafo.config.log_file
```